relay-event-normalization/src/nel.rs

```diff
@@ -0,0 +1,77 @@
+//! Contains normalization for NEL reports.
+
+use chrono::{DateTime, Duration, Utc};
+use relay_event_schema::protocol::{
+    Contexts, Event, HeaderName, HeaderValue, Headers, LogEntry, NelContext, NetworkReportRaw,
+    PairList, Request, ResponseContext, Timestamp,
+};
+use relay_protocol::Annotated;
+
+/// Normalizes the event using the provided [`NetworkReportRaw`].
+pub fn normalize(event: &mut Event, nel: Annotated<NetworkReportRaw>) {
+    // If the incoming NEL report is empty or it contains an empty body, just exit.
+    let Some(nel) = nel.value() else {
```

Comment:
I see there's a couple of `.clone()` happening below, could we prevent those by calling `into_value()` here?

Reply:
yeah, used `into_value()` here